### DISCUSSION TITLE
i can't find the host setting, the host defaut is 127.0.0,but if i use it in my server machine,i can'open it.

### DISCUSSION THREAD

**Discussion by xuyunlong123** [Arize-ai/phoenix#776] [Q&A]

**Comment by mikeldking** [comment]
Hey @xuyunlong123 thanks for trying out phoenix! So the way phoenix works is that it runs a server in a thread, so the host is wherever you are running phoenix. 

If you are using a server machine you can assign a specific port number via the `port` parameter and setup a port forward or you could use a tool to expose your local server to the internet securely. Don't have a tool in mind but there are a bunch out there.

Can you tell us a bit more about your setup? You have jupyter connected to a remote machine?

**Comment by xuyunlong123** [comment]
Thanks for your reply. My English is not good,maybe I will take up some time.
Well, I use phoenix in my server machine which I installed jupyternotebook, 
and I use a browser to visit jupyter. But  when I setup phoenix , I can't open the "http://localhost:11201/", so I  check my server machine, I found out that the program was only listening to the local 127.0.0.1 ,
 if I want use it , I must use nginx service which is not a easy way.
That's the problem I met.

**Reply by xuyunlong123** [reply]
So I want know how I can change the defaut host settings , and I suggest that the programe can listen all.

**Reply by mikeldking** [reply]
Oh I see, yes I think I get your difficulty. I'll try to reproduce and get you a solid answer soon. Thanks for trying it out! Appreciate it
